ci/jenkins/pipelines/prs/skuba-test-libvirt.Jenkinsfile

```diff
@@ -0,0 +1,166 @@
+/**
+ * This pipeline verifies on a Github PR:
+ *   - skuba unit and e2e tests
+ *   - Basic skuba deployment, bootstrapping, and adding nodes to a cluster
+ */
+
+pipeline {
+    agent { node { label 'caasp-team-private-integration-nue && kvm' } }
+
+    environment {
```

Comment:
We need to add the credentials for the `jenkins` user used to acces the KVM host. Those are not defined in the workers in general (I copied them manually to the test worker)

Reply:
I am wondering what it needs, I guess it just requires the ssh key and I am not sure how to tell libvirt provider which one to use. Gonna review if there is anything in the docs, not sure we can configure the ssh key for the server, it might use the once associated by default on the current user.

Reply:
I forgot to mention this: I was using this uri for the qemu connection in the test worker `qemu+ssh://jenkins@node52.cloud.suse.de/system` and it was working, BUT now it is not. I need to check. I remember at some point I setup the ssh-agent, but now even if I do, it is not working.

More strangely, if I test the connection with `virsh` and add the key to `ssh-agent` it works
```
$> eval (ssh-agent)
$> ssh-add /home/jenkins/.ssh/jenkins
$>virsh -c qemu+ssh://jenkins@node52.cloud.suse.de/system
```

Reply:
We can add this key-pair to jenkins as credentials and use in the same way we use other credentials.